backend/app_item/models.py

```diff
@@ -126,7 +121,7 @@ class ResourceRequest(models.Model):
     total_units = models.PositiveSmallIntegerField(_("total units"), default=0, blank=False)
     description = models.TextField(_("description"), default="", blank=True, null=False, max_length=500)
 
-    added_on = models.DateTimeField(_("added on"), auto_now_add=timezone.now)
+    added_on = models.DateTimeField(_("added on"), auto_now_add=True)
```

Comment:
`auto_now_add` should be timezone aware, so keep it `=timezone.now`

Reply:
I think it is timezone aware through the `USE_TZ=True` setting. The setting should be True/False, and the behaviour can't be customized by the value of `auto_now_add`.
https://docs.djangoproject.com/en/5.1/ref/models/fields/#django.db.models.DateField.auto_now_add

Right now, we were just "lucky" that `timezone.now` was also interpreted as `True` but the IDE was flagging the value as not being correct


![Screenshot 2025-03-10 at 12 29 34](https://github.com/user-attachments/assets/a552e8d6-3d69-4a35-85f7-8267b6e9fb42)

Reply:
:+1: